docs/quick_start/quick_start_with_datagrip.md

```diff
@@ -0,0 +1,40 @@
+# 5.Getting Started With Kyuubi and DataGrip
+## 5.1 what is DataGrip
+DataGrip is a multi-engine database environment released by JetBrains, supporting MySQL and PostgreSQL, Microsoft SQL Server and Oracle, Sybase, DB2, SQLite, HyperSQL, Apache Derby and H2.
+
+[About DataGrip](https://www.jetbrains.com/datagrip/)
+## 5.2 Preparation
+### 5.2.1 Get DataGrip And Install
+Please go to [Download DataGrip](https://www.jetbrains.com/datagrip/download) to get and install an appropriate version for yourself.
+### 5.2.2 Get Kyuubi Started
+[Get kyuubi server started](https://kyuubi.readthedocs.io/en/latest/quick_start/quick_start.html) before you try DataGrip with kyuubi.
+
+For debugging purpose, you can use ```tail -f``` or ```tailf``` to track the server log.
+## 5.3 Configurations
+### 5.3.1 Start DataGrip
+After you install DataGrip, just launch it.
+### 5.3.2 Select Database
+Substantially, this step is to choose a JDBC Driver type to use later. We can choose Apache Hive to set up a driver for Kyuubi.
+
+![select database](../imgs/dataGrip/select_database.png)
+### 5.3.3 Datasoure Driver
+You should first download missing driver files, just click on the link below, DataGrip will download and install those.
```

Comment:
You should first download the missing driver files. Just click on the link below, DataGrip will download and install those.